code/modules/admin/verbs/possess.dm

```diff
@@ -9,22 +9,25 @@
 
 	var/turf/T = get_turf(O)
 
-	if(T)
-		log_admin("[key_name(usr)] has possessed [O] ([O.type]) at ([T.x], [T.y], [T.z])")
-		message_admins("[key_name_admin(usr)] has possessed [O] ([O.type]) at ([T.x], [T.y], [T.z])", 1)
-	else
-		log_admin("[key_name(usr)] has possessed [O] ([O.type]) at an unknown location")
-		message_admins("[key_name_admin(usr)] has possessed [O] ([O.type]) at an unknown location", 1)
-
-	if(!usr.control_object) //If you're not already possessing something...
-		usr.name_archive = usr.real_name
-
-	usr.loc = O
-	usr.real_name = O.name
-	usr.name = O.name
-	usr.client.eye = O
-	usr.control_object = O
-	feedback_add_details("admin_verb","PO") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+	var/confirm = alert("Are you sure you want to possess [O]?", "Confirm posession", "Yes", "No")
+
+	if(confirm == "Yes")
```

Comment:
We prefer early returns over pointlessly changing the tab level of large blocks of code. Instead of this, you should do
```
var/confirm = alert(...)
if(confirm == "No")
    return